rosbag2_cpp/src/rosbag2_cpp/action_utils.cpp

```diff
@@ -0,0 +1,152 @@
+// Copyright 2025 Sony Group Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <cstring>
+#include <string>
+#include <regex>
+#include <unordered_map>
+
+#include "rosbag2_cpp/action_utils.hpp"
+
+namespace rosbag2_cpp
+{
+// The postfix of the action internal topics and service event topics
+const std::unordered_map<TopicsInAction, std::string> ActionTopicPostfix = {
```

Comment:
Please rename it to comply with the ROS 2 naming convention and provide more clarity on what this map represents.
```suggestion
const std::unordered_map<TopicsInAction, std::string> action_topic_to_postfix_map = {
```

Reply:
this one is still alive?